runbooks/deployCyphernodeAWS.md

```diff
@@ -0,0 +1,17 @@
+Run book how to create an online cyphernode instance
+
+1. Generate a key pair in ec2 console and download it https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#KeyPairs:
+1. connect to cloud formation https://console.aws.amazon.com/cloudformation/home?region=us-east-1
+1. deploy new stack using template file `infra\cyphernode-ec2-template.json`
+1. grab from the outputs section after the deployment is complete the public dns
+1. go from the console on a unix system where the console is located and run (replace with appropriate key name) `chmod 400 guillaume.pem && ssh -i "guillaume.pem" ubuntu@ec2-54-198-99-73.compute-1.amazonaws.com`
+1. generate an ssh key and add it to deploy keys in the git hub org.
```

Comment:
Why is this step needed? (why is github involved?)

Reply:
To authenticate to github when we need to pull the code. Won't be needed when repo is public or if we store artifacts of the repo elsewhere (ex: s3)